### DISCUSSION TITLE
Link directly to Browse > Filesystem from file info pane

### DISCUSSION THREAD

**Discussion by sevmonster** [jcorporation/myMPD#729] [Ideas]
There is a link to the out-of-player `/browse` endpoint in the file info modal, but it would be nice if there was a button to open the directory in-player with Browse > Filesystem. I have, like I am sure many people do, many untagged albums or other groups tracks organized by directory, that I cannot easily browse in any other way. There are also some filetypes mpd/myMPD does not recognize the tags for, such as videogame tracks decoded with vgmplay/GME/etc.

**Comment by jcorporation** [comment]
Good idea!

**Reply by jcorporation** [reply]
Is now implemented in the v9.3.0 branch.

**Reply by sevmonster** [reply]
Awesome, thank you!
